metricbeat/module/kubelet/container/_meta/fields.yml

```diff
@@ -1,9 +1,139 @@
 - name: container
   type: group
   description: >
-    node
+    kubelet container metrics
   fields:
-    - name: example
+    - name: name
       type: keyword
       description: >
-        Example field
+        Container name
+    - name: cpu
+      type: group
+      description: >
+        CPU usage metrics
+      fields:
+        - name: usage
+          type: group
+          fields:
+            - name: core
+              type: group
+              fields:
+                - name: ns
+                  type: long
+                  description: >
+                    Container CPU Core usage nanoseconds
+            - name: nanocores
+              type: long
+              description: >
+                CPU used nanocores
+    - name: logs
+      type: group
+      description: >
+        Logs info
+      fields:
+        - name: available
+          type: group
+          fields:
+            - name: bytes
+              type: long
+              description: >
```

Comment:
you can add here `formate: bytes` so the Kibana index pattern will contain it and will directly make sense of the unit.